### PR TITLE
chore: move onwards calls to json column

### PR DESCRIPTION
**Pull request by ljones92**

### CODE REVIEW

File: src/shared/schema/avl.schema.ts [hunk 1]
```diff
@@ -222,7 +222,7 @@ export const siriSchemaTransformed = (errors?: AvlValidationError[]) =>
                     vehicleActivity.Extensions?.VehicleJourney?.Operational?.TicketMachine?.JourneyCode ?? null,
                 vehicle_unique_id: vehicleActivity.Extensions?.VehicleJourney?.VehicleUniqueId ?? null,
                 has_onward_calls: !!vehicleActivity.MonitoredVehicleJourney.OnwardCalls,
```
Comment (DeannaSharma) [inline src/shared/schema/avl.schema.ts:224]: Can we remove the `has_onward_calls` column now? I think it was used in the code to know when to add items to the onwards calls table but I can't see any use for it in the code now.

File: src/shared/avl/utils.test.ts
Comment (DeannaSharma) [inline src/shared/avl/utils.test.ts:1]: Would it be worth updating the mock data for the `generateSiriVmAndUploadToS3` test to include some onward calls data? Just so we're covered and now its generating the siri-vm correctly?

File: src/shared/database.ts [hunk 2]
```diff
@@ -223,6 +222,17 @@ export interface AvlTable {
...
+    onward_calls: ColumnType<
+        {
+            stop_point_ref: string | null;
+            aimed_arrival_time: string | null;
+            expected_arrival_time: string | null;
+            aimed_departure_time: string | null;
+            expected_departure_time: string | null;
+        }[],
+        string,
+        string
```
[the remark this answers is not in the document]
Comment (ljones92) [reply src/shared/database.ts:234]: It needs both strings here, it's to specify the different Selectable, Insertable and Updateable types for the column

File: src/shared/schema/avl.schema.ts (hunk 1 above)
Comment (ljones92) [reply src/shared/schema/avl.schema.ts]: Removed it and added in a migration to drop the column and the avl_onward_call table

**Review by DeannaSharma — APPROVED** [review]

**Pull request merged**